packages/subgraph/src/utils/constants.ts

```diff
@@ -7,6 +7,5 @@ export const VOTER_STATE = new Map<number, string>()
   .set(2, 'Yes')
   .set(3, 'No');
 
-export const MAJORITY_VOTING_INTERFACE = '0xc52cd5d9';
-export const ERC20_VOTING_INTERFACE = '0x27a0eec0';
-export const ALLOWLIST_VOTING_INTERFACE = '0x9dd60761';
+export const ERC20_VOTING_INTERFACE = '0x6ce719bd';
+export const ALLOWLIST_VOTING_INTERFACE = '0x03e18c89';
```

Comment:
Probably not the time to open this can, but the name we've sort of agreed to is "Address List" (unless a better one appears)

Reply:
nice, i assume it will be handled by the related renaming task